src/components/organisms/teacher/OakUnitListOptionalityItemCard/OakUnitListOptionalityItemCard.tsx

```diff
@@ -79,7 +85,7 @@ export const OakUnitListOptionalityItemCard = (
     props;
 
   return (
-    <OakFlex $boxSizing={"border-box"} $flexGrow={1} role="listitem">
+    <OakLIFlex $display={"flex"} $listStyle={"none"} $flexGrow={1}>
```

Comment:
are `$display={"flex"} $flexGrow={1}` redundant since they're already styled on LN14?

FWIW, i'd probably use `<OakLI $display={"flex"} $listStyle={"none"} $flexGrow={1}>` instead of creating a new local component on LN14

Reply:
<OakLI $display={"flex"} $listStyle={"none"} $flexGrow={1}>
I tried this first but the $flexGrow didn't get added to the li.
Yes they are redundant now, I forgot to remove them when I created OakLIFlex